tests/test_cli.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from typer.testing import CliRunner
```

Comment:
this is cool! good that they have a module helping with that!

Reply:
Yeah this was a great help!